src/CMakeLists.txt

```diff
@@ -27,40 +49,105 @@ if(WIN32)
 
     set(RADARE2_INCLUDE_DIRS "${IAITO_WIN32_DIR}/radare2/include/libr" "${IAITO_WIN32_DIR}/include")
 endif()
+#
+# Win32 setting ends
 
-
-find_package(Radare2 REQUIRED)
-include_directories(${RADARE2_INCLUDE_DIRS})
-link_directories(${RADARE2_LIBRARY_DIRS})
```

Comment:
`link_directories(${RADARE2_LIBRARY_DIRS})` is still needed, because `RADARE2_LIBRARIES` not always contains the full absolute paths for the libraries. On anything except Windows, FindRadare2.cmake use pkg-config to find radare, which sets `RADARE2_LIBRARIES` to something like `r_core r_bin ...` and `RADARE2_LIBRARY_DIRS` to the library dir.